src/utils/feature.flag.ts

```diff
@@ -2,14 +2,8 @@
  * Feature flags will be determined by environment variables and all environment variables in nodejs are
  * either string or undefined. This function will ensure that 'false', '0', 'off' etc remain falsy
  */
-export const isActiveFeature = (flag: string | undefined): boolean => {
+import { FeatureFlag } from 'config';
 
-  if (flag === undefined) {
-    return false;
-  }
-  const featureFlag = flag.toLowerCase();
-  return !(featureFlag === "false" ||
-    featureFlag === "0" ||
-    featureFlag === "off" ||
-    featureFlag === "");
+export const isActiveFeature = (flag: FeatureFlag): boolean => {
```

Comment:
Nice one, but any refactoring, outside Trust work, can be please agreed with toro to avoid conflicts.

Reply:
yes, we'd like to keep refactoring to a minimum and to keep things as simple as possible as nearly everyone here has very little nodejs skills

Reply:
@mouhajer-ch 
could you please drop me full name of Toro? Thank you

@markpit 
It is good chance for ppl to learn TS
Also previous implementation do not use Typescript at all. Need to use TS power

@mouhajer-ch @markpit 
me need you decision - revert all changes or leave them with changes requested by @mwestacott ?
Thank you

Reply:
Keep this code but you need to change the params expected